src/en/config-azure.md

```diff
@@ -1,123 +1,194 @@
+
+According to all the above, the Azure section of file `environments.yaml` for
+this example would look like this (comments removed for simplicity):
 
 ```yaml
-# https://jujucharms.com/docs/config-azure.html
     azure:
         type: azure
-        # location specifies the place where instances will be started,
-        # for example: West US, North Europe.
-        #
-        location: West US
-        # The following attributes specify Windows Azure Management
-        # information. See:
-        # http://msdn.microsoft.com/en-us/library/windowsazure
-        # for details.
-        #
-        management-subscription-id: 00000000-0000-0000-0000-000000000000
-        management-certificate-path: /home/me/azure.pem
-        # storage-account-name holds Windows Azure Storage info.
-        #
-        storage-account-name: abcdefghijkl
-        # force-image-name overrides the OS image selection to use a fixed
-        # image for all deployments. Most useful for developers.
-        #
-        # force-image-name: b39f27a8b8c64d52b05eac6a62ebad85__Ubuntu-13_10-amd64-server-DEVELOPMENT-20130713-Juju_ALPHA-en-us-30GB
-        # image-stream chooses a simplestreams stream to select OS images
-        # from, for example daily or released images (or any other stream
-        # available on simplestreams).
-        #
-        # image-stream: "released"
+        location: Central US
+        storage-account-name: jujuazure
+        management-subscription-id: f717c8c1-8e5e-4d38-be7f-ed1e1c879e18
+        management-certificate-path: /home/ubuntu/.juju/juju-azure.pem
 ```
 
-This is the configuration environments.yaml file needed to run on Windows Azure.
-You will need to set relevant values for options `management-subscription-id`,
-`management-certificate-path`, and `storage-account-name`.
+Finally, switch to the Azure provider and bootstrap:
 
-**Note:** Other than `location` the other key value defaults are recommended,
-but can be updated to your preference.
-
-**Note:** Ensure that `management-certificate-path` is set to use the .pem
-file, NOT the .cer file, doing so will result in an
-[out of memory error](https://bugs.launchpad.net/ubuntu/+source/juju-core/+bug/1250007).
+```bash
+juju switch azure
+juju bootstrap --debug
+```
 
+A successful bootstrap will result in the controller being visible in the Classic console
+under 'Virtual Machines' in the left pane:
 
-## Config Values
+![bootstrap machine 0 in Azure portal](media/config-azure-stable_machine_0.png)
 
-Generate a new certificate for Juju usage (or use an existing one). Suggest to
-use a 'Common Name' with 'Juju' in it so it's obvious when viewed in the web UI
-later. On Ubuntu, run the following commands to generate a new certificate:
 
-```bash
-openssl req -x509 -nodes -days 3650 -newkey rsa:2048 -keyout azure.pem -out azure.pem
-openssl x509 -inform pem -in azure.pem -outform der -out azure.cer
-chmod 600 azure.pem
-```
+## Additional notes
 
```

Comment:
This is where I would put the stuff regarding AZs etc.
It is probably also worth mentioning that storage support is available in Azure and pointing to the storage docs for details (even though there aren't any there)